metadata/src/main/java/org/apache/kafka/image/writer/ImageWriterOptions.java

```diff
@@ -29,6 +29,7 @@
 public final class ImageWriterOptions {
     public static class Builder {
         private MetadataVersion metadataVersion;
+        private MetadataVersion orgMetadataVersion;
```

Comment:
How about "requestedMetadataVersion" ?

Reply:
Thanks. Refactored to `requestedMetadataVersion`